db/migrate/20221003172937_add_email_to_reservations.rb

```diff
@@ -0,0 +1,12 @@
+class AddEmailToReservations < ActiveRecord::Migration[7.0]
+  class Reservation < ActiveRecord::Base
+    end
+
+    def change
+      add_column :reservations, :email, :string
```

Comment:
I would add null: false here, it will cause problems if you already have reservations in database but you can simply clear all reservations if you do not need to keep them